api/v1alpha1/clusterextension_types.go

```diff
@@ -337,8 +337,8 @@ type CatalogSource struct {
 	//
 	// If unspecified, the default value is "Enforce".
 	//
-	//+kubebuilder:validation:Enum:=Enforce;Ignore
-	//+kubebuilder:default:=Enforce
+	//+kubebuilder:validation:Enum:=CatalogProvided;SelfCertified
```

Comment:
We should also update the values in the GoDoc for this field

Reply:
Updated, thanks for pointing it out 👍